cmd/antrea-controller/options.go

```diff
@@ -79,4 +79,21 @@ func (o *Options) setDefaults() {
 	if o.config.APIPort == 0 {
 		o.config.APIPort = apis.AntreaControllerAPIPort
 	}
+
+	// These defaults override the client-go defaults of 5.0 for QPS and 20
+	// for Burst (https://pkg.go.dev/k8s.io/client-go@v0.21.1/rest#Config).
+	// These values will be used for all the Controller's clientsets, and in
+	// particular the CRD clientset. We have observed that these defaults
+	// work much better for the CRD mirroring controller when Antrea is
+	// upgraded in a cluster with a large number of existing Antrea-native
+	// policies. In this scenario, the mirroring controller needs to mirror
+	// many legacy (*.antrea.tanzu.vmware.com) CRDs to their crd.antrea.io
+	// replacement, which will take a very long time with client-go's QPS
+	// and Burst defaults.
```

Comment:
will we revert these values one day when we no longer need to mirror because we have crossed the min num of releases to support the older APIs?

Reply:
We could. I don't believe there is too much downside to having these higher limits, since this is for the controller only and not for each agent.